.github/workflows/cd.yaml

```diff
@@ -0,0 +1,24 @@
+name: CD
```

Comment:
Nit: Should this be `publish` or `package` or something like that?  I wouldn't really call this continuous or a deployment 😆

Reply:
If we had access to the testpypi Zappa package it would be somewhat more continuous at least 😬